appveyor.yml

```diff
@@ -0,0 +1,41 @@
+#TODO: Change version
+version: '0.0.1.{build}'
+
+configuration:
+  - Release
+  - Debug
+
+platform:
+  - x86
+  - x64
+
+environment:
+  matrix:
+  - GENERATORS: "Visual Studio 15 2017"
+    APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+
+  - GENERATORS: "Visual Studio 14 2015" 
+  
+  - GENERATORS: "Visual Studio 12 2013"
+  
+  - GENERATORS: "Visual Studio 11 2012"
+   
+  - GENERATORS: "Visual Studio 10 2010"
+
+  - GENERATORS: "MSYS Makefiles"
```

Comment:
Move vote for singular `GENERATOR` unless I'm missing a good reason for plural here.